### DISCUSSION TITLE
Downloader Timing Out

### DISCUSSION THREAD

**Discussion by aerconos** [Avnsx/fansly#131] [General]
Any tips for troubleshooting a downloader that timed out? I started to receive a bunch of connection errors. When I restarted the .exe downloader, it times out. I get this message, then nothing else:

 Info | 14:38 || Reading config.ini file ...

 info | 14:38 || Username validation successful!
 Info | 14:38 || Deduplication is automatically enabled for;
                 C:\Users\Home\Downloads\Fansly_Downloader_0.4.1-Windows\nameofmodelhere

Any ideas on what's going on?

**Comment by Avnsx** [comment]
Can you further explain what you mean exactly by 'timed out'?

Is it just staying open and doing nothing or does it entirely crash and close out?

Since it got stuck at hashing,  would it be possible that it's lacking neccesarry permission to read the previous download folder?

Try right clicking fansly downloader.exe > properties > general tab; at the bottom it might say something about 'trust this program' > tick that. and also somewhere it should say start as admin, try ticking that too.

Next go to the previous download folder > right click > permissions > security > tick full control for all user accounts.

If your previous download folder contains alot of content and your hardware is slow it might be possible that it's hashing and might just need some time.

By the way, does this also happen when you try to not update a previous download folder, but instead start a entirely fresh downloading session?

**Reply by aerconos** [reply]
### Some context
I just downloaded the software for the first time yesterday after I discovered it on a reddit forum. I followed the instructions and installed the .exe file, including modifying the config file. The program began scraping as intended, but after about 30-40 minutes, it said it had a bunch of connection errors and stopped working.

### In response

> Can you further explain what you mean exactly by 'timed out'?

I restarted the program thinking it would pick up where it left off. After the 3 lines of dialogue I posted in my original post, nothing happens. I am assuming this to be the program 'timing out'. I tried changing the parameters to just timeline, or messages, but that had no effect. 

> Since it got stuck at hashing, would it be possible that it's lacking neccesarry permission to read the previous download folder?

I'm not sure when or how the permissions would be changed, as it was working fine, then just sort of stopped. 

> Try right clicking fansly downloader.exe > properties > general tab; at the bottom it might say something about 'trust this program' > tick that. and also somewhere it should say start as admin, try ticking that too.
> 
> Next go to the previous download folder > right click > permissions > security > tick full control for all user accounts.

I double-checked all of the permissions are checked. 

> If your previous download folder contains alot of content and your hardware is slow it might be possible that it's hashing and might just need some time.

There is only about 60GB in the folder and plenty of drive space left. 

> By the way, does this also happen when you try to not update a previous download folder, but instead start a entirely fresh downloading session?

I have not tried this yet, but that was going to be my next thing to try.

**Reply by aerconos** [reply]
Update: Deleting the previous folder and re-running the program worked. Thank you.
